test/Mono.Linker.Tests.Cases/Attributes.Debugger/DebuggerDisplayAttributeOnAssemblyUsingTarget.cs

```diff
@@ -13,8 +13,13 @@ namespace Mono.Linker.Tests.Cases.Attributes.Debugger {
 	// Can be removed once this bug is fixed https://bugzilla.xamarin.com/show_bug.cgi?id=58168
 	[SkipPeVerify (SkipPeVerifyForToolchian.Pedump)]
 	
+#if NETCOREAPP
+	[KeptMemberInAssembly ("System.Private.CoreLib.dll", typeof (DebuggerDisplayAttribute), ".ctor(System.String)")]
```

Comment:
Add Corlib constant instead of such #if(s)

Reply:
There are some tests that look for types that used to be in mscorlib but now live in 'System.Collections', for example (see this commit: https://github.com/mono/linker/pull/489/commits/36f5e266ea9aa56c395883c4c4787c3bfaac92fc), so this isn't just a matter of replacing "mscorlib" with "System.Private.CoreLib" in all cases.

If you prefer, I could introduce a constant for the cases where the type is in mscorlib/System.Private.CoreLib and use `#if NETCOREAPP` for the rest, but that seems a bit inconsistent to me.

I'd prefer to either
- use the explicit assembly names in attributes, OR
- introduce an abstraction that knows where the expected types live on mono vs .NET Core, and remove the assembly name from these attributes.

Let me know what you think - happy to follow your suggestion (or @mrvoorhe's) on this.

Reply:
Please introduce the constant for corlib name, there is no inconsistency in tests what matters is the readability

Reply:
I agree constant for corlib and then anything else will have to use #if's and duplicate the asserts.

Reply:
The testcases get compiled both all at once into a single dll, and individually at test run time, so I need to make sure the reference containing the constant is defined in something referenced at test run time. Does Mono.Linker.Tests.Cases.Expectations seem like a reasonable place to put it @mrvoorhe?

Reply:
Yes.  Add some new static class in that assembly

Reply:
Done in https://github.com/mono/linker/pull/489/commits/29fe2d89f6f2dbf8f518f8c03cf5a3e5b8a80dae and https://github.com/mono/linker/pull/489/commits/dac1eaaaeccf3d82937e68814018480d871f097d.